circle.yml

```diff
@@ -40,7 +39,10 @@ machine:
 
 checkout:
   post:
-    - git clone --depth 1 -b ${ST2_PACKAGES_BRANCH} ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
+    - |
+      git clone --depth 1 ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
+      cd /home/ubuntu/st2/st2-packages
+      git checkout ${CIRCLE_BRANCH} || true
```

Comment:
^^ 
Yeah, initially, we couldn't think seriously about such option at all, because we couldn't touch entire existing `st2` codebase and release machinery with our `st2-packages` changes.
So we kept that package logic isolated in new repo to avoid overflowing other team members (especially considering we had a lot of experiments and pretty big code flow).

When `st2-packages` matures and stabilizes, it's good to consider pros/cons again for keeping it in one or separated repos :+1: